exporter/prometheusexporter/collector.go

```diff
@@ -38,7 +41,7 @@ func newCollector(config *Config, logger *zap.Logger) *collector {
 		logger:         logger,
 		namespace:      sanitize(config.Namespace),
 		sendTimestamps: config.SendTimestamps,
-		constLabels:    config.ConstLabels,
+		constLabels:    prometheus.Labels(config.ConstLabels),
```

Comment:
I don't think this is necessary, [config.ConstLabels is already of type prometheus.Labels](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/prometheusexporter/config.go#L37)